migrations/2019-02-19-141312_agg_generations/up.sql

```diff
@@ -0,0 +1,9 @@
+create materialized view if not exists public.agg_generations as
```

Comment:
There is another way to make a change in a migration--this will only take effect when the DB is recreated, and we don't really want to do that.

Reply:
@johnsnewby removed the materialized view. the query as a whole seems fast enough to me and also for large dataset/range pagination should help. 
Someone using endpoint for the large dataset/range without pagination options provided I believe is already aware of the implications(reduced speed).